be/src/cloud/cloud_base_compaction.cpp

```diff
@@ -48,8 +48,16 @@ CloudBaseCompaction::CloudBaseCompaction(CloudStorageEngine& engine, CloudTablet
 CloudBaseCompaction::~CloudBaseCompaction() = default;
 
 Status CloudBaseCompaction::prepare_compact() {
+    Status st;
+    Defer defer_set_st([&] {
+        cloud_tablet()->set_last_base_compaction_status(st.to_string());
```

Comment:
If the status code is OK, the compaction will continue to execute. Do I need to set the status code at this time?